docs/sources/panels-visualizations/query-transform-data/transform-data/index.md

```diff
@@ -135,6 +135,13 @@ Use this transformation to add a new field calculated from two other fields. Eac
     - **Natural logarithm (ln)** - Returns the natural logarithm of a given expression.
     - **Floor (floor)** - Returns the largest integer less than or equal to a given expression.
     - **Ceiling (ceil)** - Returns the smallest integer greater than or equal to a given expression.
+  - **Cumulative functions** - Apply functions on the current row and all preceding rows.
```

Comment:
Don't forget, we now update the transformation docs content here (public/app/features/transformers/docs/content.ts), and auto-build this markdown as described in the aforementioned module. :)

Reply:
This should be fixed now!